code/game/objects/effects/decals/turfdecal/tilecoloring.dm

```diff
@@ -21,52 +21,283 @@
 
 #undef PRIDE_ALPHA
 
+/// Blue tiles
+
 /obj/effect/turf_decal/tile/blue
 	name = "blue corner"
+	var/color_name = "blue"
```

Comment:
wtf are u doing

Reply:
Thing from before second commit. Forgot to take it out, it was late.